src/types.ts

```diff
@@ -56,6 +56,18 @@ export enum IconColorType {
     CONTRAST = 'contrast',
 }
 
+export interface CustomTile {
+    id: string
+    displayName: string
+    linkAddress: string
```

Comment:
Navnet `linkAddress` kunne kanskje vore endra til noko litt meir generelt som passar for både link og bilde, f.eks. `sourceUrl`